### PR TITLE
[wasm][debugger] Trying to fix MT debugging experience.

### PR DESCRIPTION
**Pull request by thaystg**
I'm doing this workaround to avoid memory allocation while debugging a MT wasm app, it looks like there is some failure while updating the memory in all the workers while some of them are paused.
This will fix the CI and also improve the customer experience.

### CODE REVIEW

**Comment by thaystg** [comment]
First try passed the MT debugger tests.

File: src/mono/wasm/runtime/debug.ts [hunk 1]
```diff
@@ -149,6 +151,11 @@ export function mono_wasm_debugger_attached(): void {
     if (runtimeHelpers.waitForDebugger == -1)
         runtimeHelpers.waitForDebugger = 1;
     cwraps.mono_wasm_set_is_debugger_attached(true);
+    if (MonoWasmThreads) {
+        const wasmMemory = Module.getMemory();
+        wasmMemory.grow(wasmMemory.buffer.byteLength >>> 16);
```
Comment (pavelsavara) [inline src/mono/wasm/runtime/debug.ts:156]: I'm not sure how this is related to debugger or threads. Could you please elaborate ?

**Pull request closed without merging**